Prose/ProseLib/Sources/ConversationFeature/Chat/MessageView.swift

```diff
@@ -69,34 +45,74 @@ public struct MessageView: View {
     }
 }
 
+public struct MessageViewModel: Equatable {
```

Comment:
No, we will not have ViewModels anymore, this was code from before the migration to TCA 😉 I still need to migrate it, but since it's a screen that will be replaced by a WebView, I have'nt prioritized it.